FAQ/FAQ-Synchronization.md

```diff
@@ -17,7 +17,21 @@ It may seem tedious, but it's a critical step of running your own full node and
 
 ## Can I skip the synchronization?
 
-You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use [FastSync](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to synchronize your node faster. You can also follow [this video](https://youtu.be/VNMnd-dX9Q8?t=1730). However, be sure to read the Fast Sync document to understand the potential trust issues involved with Fast Sync. 
+You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use FastSync to synchronize your node faster. Be sure to [read this FastSync document](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to understand the potential trust issues involved with FastSync.
+
+To use FastSync, [ssh into](FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) your BTCPayServer instance and run the following commands:
+```
+sudo su -
+cd /root/btcpayserver-docker/
```

Comment:
Have you tried these commands?
For me:
`cd /root/btcpayserver/btcpayserver-docker` does not work. You don't even need that step

This works
```
sudo su -
./btcpay-down.sh
cd contrib
cd FastSync
./load-utxo-set.sh
then ./btcpay-up.sh
```

Reply:
Better should be `cd $BTCPAY_BASE_DIRECTORY/btcpayserver-docker/`